app.js

```diff
@@ -2,13 +2,17 @@ const express = require('express');
 const cors = require('cors');
 const morgan = require('morgan');
 
+const { globalErrorhandler } = require('./utils/error');
+
 const createApp = () => {
   const app = express();
 
   app.use(express.json());
   app.use(cors());
   app.use(morgan('tiny'));
 
+  app.use(globalErrorhandler);
+
```

Comment:
```suggestion
  app.use(morgan('tiny'));
  //app.use(route);

  app.all('*', (req, res, next) => {
    const err = new Error(`Can't fine ${req.originalUrl} on this server!`);

    err.statusCode = 404;

    next(err);
  });

  app.use(globalErrorHandler);
```
morgan과 globalErrorhandler 미들웨어 사이에 다음과 같이 routes로 처리 되지 않는 엔드포인트에 대해 일괄적으로 핸들링하는 코드가 추가되면 좋을 것 같습니다.